lib/ios-crash-log.js

```diff
@@ -0,0 +1,68 @@
+import path from 'path';
+import _fs from 'fs';
+import _ from 'lodash';
+import glob from 'glob';
+import { getLogger } from 'appium-support';
+import B from 'bluebird';
+
+let fs = {
```

Comment:
yeah both of these are now in appium-support, so you can just `import { fs } from 'appium-support'`, and then use `await fs.stat` etc as necessary.

also a note about this code: you're promisifying the sync version of stat, which probably won't do what you had in mind!